sklearn/cluster/spectral.py

```diff
@@ -18,8 +18,8 @@
 from .k_means_ import k_means
 
 
-def discretize(vectors, copy=True, max_svd_restarts=30, n_iter_max=20,
-               random_state=None):
+def discretize(vectors, max_svd_restarts=30, n_iter_max=20,
+               random_state=None, copy=True):
```

Comment:
In order to stay backward compatible, we cannot easily change the parameter order. So please revert this change